src/main/groovy/com/calgaryscientific/gradle/VeracodeDetailedReport.groovy

```diff
@@ -142,7 +142,7 @@ class VeracodeDetailedReport {
     static List<Node> filterOpenFlaws(List<Node> flaws) {
         flaws.findAll { flaw ->
             String status = flaw.attribute('remediation_status')
-            (status == "Open" || status == "New" || status != "Fixed")
+            (status == "Open" || status == "New" || (status != "Fixed" && status != "Mitigated"))
```

Comment:
what does this Mitigated mean? openFlawsAndMitigated?

Reply:
Mitigated means that the flaw found by Veracode has counter measures in place that makes it not exploitable. So the flaw doesn't need to show up in the Open flaw list.

The name still applies to Open flaws, however, I made it so it shows the flaws that don't have a known status so I can catch new statuses that are not accounted for, that is how the Mitigated status showed up in my report when before I only knew about Fixed, Open and New.